From 01fe3b241b00cde8c5e249928ae990491ad74fa9 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Mon, 3 Feb 2014 16:30:31 +0100
Subject: [PATCH 171/212] usb: add microsoft os descriptors compat property

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1391445032-5540-5-git-send-email-kraxel@redhat.com>
Patchwork-id: 57074
O-Subject: [RHEL-7 qemu-kvm PATCH 4/5] usb: add microsoft os descriptors compat property
Bugzilla: 1039530
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

Turn off for rhel6 machine types.  RHEL only patch.

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
---
 hw/i386/pc_piix.c | 4 ++++
 1 file changed, 4 insertions(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/i386/pc_piix.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/hw/i386/pc_piix.c b/hw/i386/pc_piix.c
index 2a6ac6d..db5f2e0 100644
--- a/hw/i386/pc_piix.c
+++ b/hw/i386/pc_piix.c
@@ -1074,6 +1074,10 @@ static QEMUMachine pc_machine_rhel700 = {
         .driver   = "virtio-net-pci",\
         .property = "romfile",\
         .value    = "rhel6-virtio.rom",\
+    },{\
+        .driver   = TYPE_USB_DEVICE,\
+        .property = "msos-desc",\
+        .value    = "no",\
     }
 
 static void pc_compat_rhel650(MachineState *machine)
-- 
1.7.1

